flink-runtime/src/test/java/org/apache/flink/runtime/state/StateBackendMigrationTestBase.java

```diff
@@ -480,6 +482,14 @@ private void testKeyedMapStateUpgrade(
             // make sure that reading and writing each key state works with the new serializer
             backend.setCurrentKey(1);
             Iterator<Map.Entry<Integer, TestType>> iterable1 = mapState.iterator();
+
+            List<Map.Entry<Integer, TestType>> actualList =
```

Comment:
1. Could you put them into a seprate method, like `sortedIterator` ?
2. a minor suggestion: Use TreeMap to avoid manual sort